packages/cli/src/__tests__/cli-export.test.ts

```diff
@@ -192,6 +196,19 @@ test('should throw error when missing api key', async () => {
   await expect(
     createCli().parseAsync(['node', 'j1', 'export']),
   ).rejects.toThrow(
-    /Missing apiKey! Set the JUPITERONE_API_KEY environment variable or supply the --api-key option./,
+    /Missing option! Set the JUPITERONE_API_KEY environment variable or supply the --api-key option./,
+  );
+});
+
+test('should throw error when missing api key', async () => {
```

Comment:
```suggestion
test('should throw error when missing account', async () => {
```

Reply:
This looks to still be checking the api key, and there is another expectation below that provides the `--api-key` and throws for the missing `--acount`.